src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const prop of transforms) {
```

Comment:
what is prop? do not use short forms
```suggestion
  for (const transform of transforms) {
```